pkg/grpc/device_definition.proto

```diff
@@ -241,6 +241,31 @@ message GetDeviceStyleByIDRequest {
 	string id = 1;
 }
 
+message GetDeviceCompatibilityListRequest {
+	string make_id = 1;
+	string integration_id = 2;
+	string region = 3;
+}
+
+message Feature {
```

Comment:
I wonder if we should make `Feature` an embedded type of DeviceCompatibilities since it only lives in the context of that. The drawback is that it makes the generated type name longer. Not sure what you guys think.

Reply:
I think this is a good idea, it will definitely make things easier to follow

Reply:
I think embedding some of these would assuage my concerns about the names being not descriptive enough. Too late now, I suppose.